tools/policy-gen/protoc-gen-kumapolicy/helpers.go

```diff
@@ -31,6 +31,11 @@ func generateHelpers(
 		}
 	}
 
+	// workaround for now for policies that do not have from and to
```

Comment:
workaround so that code compile, will be replaced by a proper solution when implementing xds generation for MeshTrace

Reply:
Instead of a workaround could we just fix this for good?

Reply:
tbh I don't know how this would look like. I will "stumble" upon the solution once I start implementing the xds generation, this is just so that the code compiles. Can it wait for the XDS PR or do you want me to do this now?

Reply:
@jakubdyszkiewicz - WDYT?

Reply:
not sure if this is a workaround. The helper is currently for generating from and to methods. I think it makes sense to skip it completely if the policy is neither "from policy" nor "to policy".

Reply:
Maybe, I thought that there might be more implem needed when the xds part is implemented. I'm removing the comment and will introduce any changes if needed during xds PR.